### ISSUE TITLE
Hyperparameter for training SEED Tokenizer

### ISSUE THREAD

**Issue by Cheolhyun-Mun** [AILab-CVC/SEED#16]
Hi!
Thank you for the wonderful works.

I wonder that you can provide the detailed information for training SEED Tokenizer.
I cannot find the hyperparameter for training SEED Tokenizer in your paper.

Also, I have another question.
In the paper, SEED Tokenizer training is divided into two stages. Does that mean the Q-former is pre-trained in stage 1 and then the Q-former, codebook, decoder, and MLP are trained in stage 2?

Thank you.

**Comment by geyuying** [comment]
We have released the training code of SEED-LLaMa, including SEED tokenizer, Multimodal LLM pretraining and instruction tuning. Our Multimodal LLM training codebase supports 1. large-scale multi-node training with deepspeed 2. highly-efficient multiple training datapipes.
